src/components/v2/devtronStackManager/devtronStackManager.scss

```diff
@@ -0,0 +1,113 @@
+.stack-manager {
+    display: grid;
+    grid-template-rows: 48px 1fr;
+    grid-template-columns: 250px 1fr;
+    height: 100vh;
+
+    &.full-view-mode {
+        grid-template-columns: 100%;
+
+        &.white-background {
+            background-color: white !important;
+        }
+    }
+
+    .page-header {
+        grid-column: 1 / -1;
+    }
+
+    .stack-manager__navigation {
+        padding: 24px 12px;
+        border-right: solid 1px #d6dbdf;
+        background-color: white;
+
+        a {
+            font-size: 13px;
```

Comment:
if possible please use the CSS formulae

Reply:
Yes, will try. 
Actually, some of the things were getting overridden by global CSS so had to use specific selectors.

Reply:
@vivek-devtron Tried this but global CSS is overriding the styling so keeping it as is.